chia/util/cpus.py

```diff
@@ -0,0 +1,13 @@
+import os, multiprocessing
+
+def get_available_cpus():
+    cpu_count = multiprocessing.cpu_count()
+    try:
+        cpu_count = os.sched_getaffinity(0)
+    except AttributeError:
+        pass
+
+    if os.name == "nt" and cpu_count > 61:
+        cpu_count = 61      # Windows Server 2016 has an issue https://bugs.python.org/issue26903
```

Comment:
Looks you can spare some magic and alternatively do
```python
def get_available_cpus() -> int:
    try:
        cpu_count = os.sched_getaffinity(0)
    except AttributeError:
        cpu_count = multiprocessing.cpu_count()

    # Note: Windows Server 2016 has an issue https://bugs.python.org/issue26903
    if os.name == "nt":
        MAX = 61
        cpu_count = min(cpu_count, MAX)

    return cpu_count
```

Reply:
Good suggestion, I've included it in the change.